packages/components/molecules/f-skeleton-loader/src/components/skeletons/Rating.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div
+        :class="$style['c-rating']">
+        ★★★★★★
```

Comment:
I didn't know we can you star images like this :)

Reply:
Yea I was looking at SVGs etc but these work perfectly in this case.